packages/cli/src/credentials/credentials.service.ee.ts

```diff
@@ -25,6 +25,36 @@ export class EECredentialsService extends CredentialsService {
 		return { ownsCredential: true, credential };
 	}
 
+	/**
+	 * Retrieve the sharing that matches a user and a credential.
+	 */
+	static async getSharing(
+		user: User,
+		credentialId: number | string,
+		relations: string[] = ['credentials'],
+		{ allowGlobalOwner } = { allowGlobalOwner: true },
+	): Promise<SharedCredentials | undefined> {
+		const options: FindOneOptions = {
```

Comment:
Prevent `@typescript-eslint/no-unsafe-member-access` below.
```suggestion
		const options: FindOneOptions<SharedCredentials> & { where: ObjectLiteral } = {
```